genai-on-vertex-ai/gemini/model_upgrades/instruction-following/promptfoo/README.md

```diff
@@ -0,0 +1,50 @@
+# RAG
+### _Eval Recipe for model migration_
+
+This Eval Recipe demonstrates how to compare performance of an Instruction Following prompt with Gemini 1.5 Flash and Gemini 2.0 Flash using an unlabeled dataset and open source evaluation tool [Promptfoo](https://www.promptfoo.dev/).
+
+
+- Use case: Instruction Following 
+
+- Evaluation Dataset is based on [Instruction Following Evaluation Dataset](https://github.com/google-research/google-research/blob/master/instruction_following_eval/data/input_data.jsonl). It includes 15 randomly sampled prompts in a JSONL file `dataset.jsonl`. Each record in this file includes 1 attribute wrapped in the `vars` object. This structure allows Promptfoo to specify the variables needed to populate prompt templates (document and question), as well as the ground truth label required to score the accuracy of model responses:
+    - `prompt`: The task with specific instructions provided
+
+- Prompt Template is a zero-shot prompt located in [`prompt_template.txt`](./prompt_template.txt) with one prompt variable (`prompt`) that is automatically populated from our dataset.
+
+- [`promptfooconfig.yaml`](./promptfooconfig.yaml) contains all Promptfoo configuration:
+    - `providers`: list of models that will be evaluated
+    - `prompts`: location of the prompt template file
+    - `tests`: location of the labeled dataset file
+    - `defaultTest`: defines the scoring logic:
+        `type: answer-relevance` The answer-relevance assertion evaluates whether an LLM's output is relevant to the original query. It uses a combination of embedding similarity and LLM evaluation to determine relevance..
+        `value: "Check if the response adheres to the instructions in the prompt"` instructs Promptfoo to verify and score based on how well the generated response is aligned with the original prompt.
+        `threshold: 0.5` Mark any responses with a score below 0.5 as a failure.
+
+
+
+## How to run this Eval Recipe
```

Comment:
Please check the multi-turn chat use case to add instructions for "Launch in Cloud Shell"